QuadRelate.Helpers/BoardHelper.cs

```diff
@@ -20,6 +20,78 @@ public static List<int> AvailableColumns(this Board board)
             return availableColumns;
         }
 
+        public static bool IsGameOver(this Board board)
+        {
+            // Check for horizontal wins
+            for (var y = 0; y < Board.Height; y++)
```

Comment:
This is a large method.
You have a comment: **// Check for horizontal wins.**
Perhaps that's a clue?
Do you need another hint?

Reply:
That should be enough of a hint (I hope)